toDoList/cmd/main.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type Event struct {
+	Id          int32
+	Title       string
+	Description string
+}
+
+var DB = make(map[int32]Event)
+
+func main() {
+	router := mux.NewRouter()
+	router.HandleFunc("/", Home)
+	router.HandleFunc("/api/v1/events", GetAllEvents).Methods("GET")
+	router.HandleFunc("/api/v1/events", CreateEvent).Methods("POST")
+	router.HandleFunc("/api/v1/events", EditEvent).Methods("PUT")
+	router.HandleFunc("/api/v1/events", DeleteEvent).Methods("DELETE")
+
+	http.ListenAndServe(":8080", router)
+
+}
+
+func Home(w http.ResponseWriter, r *http.Request) {
+	fmt.Println("Welcome to the ToDo app instructions")
```

Comment:
```suggestion
	fmt.Println(`Welcome to the ToDo app instructions
	APIS: 
	
	`)
```

you can do something like that so you don't call printing method so many times